Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/Isotropic_remeshing/remesh_impl.h

```diff
@@ -494,13 +586,17 @@ namespace internal {
           Halfedge_status snew = (is_on_patch(hnew) || is_on_patch_border(hnew))
             ? PATCH
             : MESH;
+          // AF: I do not understand why we would call halfedge_added with snew = MESH
+          //     as set_status(,MESH) erases from the set, and status(h) returns MESH 
+          //     in case h is not found
+          // JT: We have to pay attention when recycling faces with potentially a wrong status
```

Comment:
right.
```
          // AF: I do not understand why we would call halfedge_added with snew = MESH
          //     as set_status(,MESH) erases from the set, and status(h) returns MESH 
          //     in case h is not found
          // JT: We have to pay attention when recycling faces with potentially a wrong status
```